src/main/resources/config/application-dev.yml

```diff
@@ -107,6 +107,8 @@ jhipster:
 # Properties to be exposed on the /info management endpoint
 info:
     guided-tour:
+        # Names of the tutorial groups that will be automatically added to newly created users
+        course-group-students: 'artemis-artemistutorial-students'
```

Comment:
I assume this needs to be added to the documentation as well: https://github.com/ls1intum/Artemis/blob/ad6300c68b1ea0174d92497a54be51305e38731f/docs/dev/guided-tour.rst#extend-configuration-file

Reply:
Good point, thank you!

I've now integrated the new property in the documentation with https://github.com/ls1intum/Artemis/pull/2245/commits/43dfa49eac75ce68b4ef03baf428e4b6c014951c